rubicon/objc/__init__.py

```diff
@@ -3,6 +3,7 @@
 from .objc import (
     objc, send_message, send_super,
     get_selector,
+    SEL, objc_id, Class, IMP, Method, Ivar, objc_property_t,
```

Comment:
Oh, I guess I misunderstood... It was confusing enough that I looked elsewhere in the changeset and somehow decided that `# Selectors end in a colon` is where same convention was ultimately used.

To be honest I'm still not sure whether I got it right or wrong.